### ISSUE TITLE
Color_Brightness Control has no effect on addressable SK6812 Color Channels

### ISSUE THREAD

**Issue by tk4arts** [esphome/issues#2413]
### The problem

Hello there,
I have a Wemos D1 Mini with ESP Home flashed to use it as a controller for my SK6812 individually adressable LEDs. Everything is working like a charm except for one issue I have. The color_brightness slider in Home Assistant is not changing anything. The slider for the white brightness is working and the overall brightness slider as well. With this configuration it is not possible to have only the white channel of the LEDs on while the others are off. Unfortunately this is what I need and what is technically possible by the SK6812 and the D1 Mini. It just feels like ESP Home does not do anything with the value from this slider.
If one of the developers could try and fix the lose end between the color_brightness slider and the neopixelbus integration that would be great. 
Thanks in advance!

### Which version of ESPHome has the issue?

2021.8.2

### What type of installation are you using?

Home Assistant Add-on

### Which version of Home Assistant has the issue?

2021.8.2

### What platform are you using?

ESP8266

### Board

Wemos D1-mini

### Component causing the issue

neopixelbus

**Comment by oxan** [comment]
I don't have a SK6812, but I can't reproduce this with my LED strips. Are you sure you're running 2021.8? This was a problem with earlier versions but should be resolved now. 

In any case we're going to need more information, including at least your config and logs, to say anything sensible about this.

**Comment by tk4arts** [comment]
Thanks for the reply. I will try to gather the needed information.

Below is a screenshot of my esphome addon page in homeassistant. That says 2021.8.2 so I am pretty sure this is the version I am running.
![image](https://user-images.githubusercontent.com/84267222/133462701-132b2e7e-46cc-4835-9367-8c57e88f04bb.png)

I now managed to get it working like it should with no effect running. 
The problem is that I want to run a custom lambda effect.
To set the color of the Leds I used:
`it.all() = current_color;`

And then the color brightness slider is not working.
Is there another way to access the currently set color and apply it to individual leds or the complete strip?

Thanks in advance!
Till
